php-practice_1.php

```diff
@@ -44,44 +43,44 @@
 
 
 // Q6 配列
-$tokyoarea = ['東京都', '埼玉県', '群馬県','栃木県','千葉県','茨城県','神奈川県'];
-echo ''.$tokyoarea[3].'と'.$tokyoarea[4].'は関東地方の都道府県です。';
+$tokyoarea = ['東京都' , '埼玉県' , '群馬県' , '栃木県' , '千葉県' , '茨城県' , '神奈川県'];
```

Comment:
[must]
カンマの前後ではなく、後ろだけ空白を入れれば良いです！
```suggestion
$tokyoarea = ['東京都', '埼玉県', '群馬県', '栃木県', '千葉県', '茨城県', '神奈川県'];
```

Reply:
[修正しました](https://github.com/nayutantan/php-practice/pull/1/commits/cd4d488f50de5034d5eda9ac64d33634e9250ba7)